src/common/src/lib.rs

```diff
@@ -35,8 +35,8 @@ pub const HYPERVISOR_HASH_INFO: Option<&'static str> = if let Some(s) = option_e
 pub const COMPILER_INFO: Option<&'static str> =if let Some(s) = option_env!("RUSTC_VERSION") && s.len() != 0 {Some(s)}else{None};
 /// The path of hypervisor_kernel
 pub const HYPERVISOR_PATH: &'static str = "\\EFI\\BOOT\\hypervisor_kernel";
-/// The path of EL1 u-boot
-pub const U_BOOT_PATH: &'static str = "\\EFI\\BOOT\\u-boot.bin";
```

Comment:
Just to confirm, these lines are unnecessary because of #5, right?

Reply:
Yes
U_BOOT_PATH became needless now